llvm/lib/CodeGen/ReachingDefAnalysis.cpp

```diff
@@ -529,14 +612,21 @@ ReachingDefAnalysis::getLocalLiveOutMIDef(MachineBasicBlock *MBB,
                                           MCRegister PhysReg) const {
   LiveRegUnits LiveRegs(*TRI);
   LiveRegs.addLiveOuts(*MBB);
-  if (LiveRegs.available(PhysReg))
+  if (Register::isPhysicalRegister(PhysReg) && LiveRegs.available(PhysReg))
```

Comment:
If `PhysReg` isn't `isPhysicalRegister`, then is there something wrong with the naming here? Or is this statement always true and can be dropped?